main.go

```diff
@@ -9,6 +9,7 @@ import (
 )
 
 func main() {
+	config.CheckUpdates()
```

Comment:
Don't use `log`. It adds the time stamp which is not necessary and makes the output harder to read. Time stemp is useful when we want to preserve the logs so we can know when it happened. This is not that case. So please use `fmt`.